src/services/authService.ts

```diff
@@ -115,10 +115,29 @@ class authService {
     return updatedUser;
   }
 
-  async updateProfilePicture(image: any) {
-    const result = await uploadCloudImage(image, null);
-
+  //####################################################
+  async updateProfilePicture(
+    image: string | undefined,
+    folder: string | undefined,
+    email: string
+  ) {
+    const result = await uploadCloudImage(image, folder);
```

Comment:
the folder should be decided by the developer, remove folder fromt he request schema